packages/ui/src/common/components/ModalLink.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { ReactNode } from 'react-markdown'
+import { Link } from 'react-router-dom'
```

Comment:
This should be "our" link (`<a>` element, not the one from the react router):

```jsx
import { Link } from '@/common/components/Link'
```

However, the `Link` type must be upgraded to:

```jsx
import React, { AnchorHTMLAttributes } from 'react'
import styled from 'styled-components'

import { Colors, Transitions } from '../constants'

interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
  dark?: boolean
  size?: 'm' | 'l'
}
```